README.md

```diff
@@ -103,6 +103,18 @@ After the above requirements have been met:
 
 That's it!
 
+### Docker
+
+If you have [Docker](https://www.docker.com/) already installed on your machine, you can use our `docker-compose.yml` to setup your project.
+
+1. Make sure you have the project cloned.
+2. Setup the following three variables in the `Dockerfile` file, see the [Twilio Account Settings](#twilio-account-settings).
```

Comment:
Can you try if you can use the existing `.env` for docker-compose.yml using [env_file](https://docs.docker.com/compose/environment-variables/#the-env_file-configuration-option) configuration? if that works we can get rid of this step.

Reply:
@po5i It seems like Docker does not accept an env file with the keyword EXPORTS, so I've moved the app variables to the app container and updated the Readme in the Docker steps. Could you please have a look? :)

Reply:
Changes look good! Thanks